README.md

```diff
@@ -23,6 +23,12 @@ $ make build-timbot
 $ make run-timbot
 ```
 
+## Packages
+
+To install packages run:
+
+`$ pip install -r requirements.txt`
```

Comment:
This isn't needed, you should instead add the numpy package to the Dockerfile.

Reply:
Going to leave for non-containerized deployments